src/core/src/sys/compute.rs

```diff
@@ -201,7 +200,7 @@ impl Compute for Virial {
                 let nj = composition[&j] as f64;
                 let potentials = system.interactions().pairs(i, j);
                 for potential in potentials {
-                    virial -= 2.0 * PI * ni * nj * potential.tail_virial() / volume;
+                    virial += 2.0 * PI * ni * nj * potential.tail_virial() / volume;
```

Comment:
This should be in a separated commit (commits should do only one thing), and maybe with an explanation message (the commit message) and associated tests.

It is fine for this one, can you try to do it for your next commits?